pkg/cri/store/image/image.go

```diff
@@ -134,17 +141,38 @@ func getImage(ctx context.Context, i containerd.Image) (*Image, error) {
 
 	id := desc.Digest.String()
 
-	spec, err := i.Spec(ctx)
+	rb, err := content.ReadBlob(ctx, i.ContentStore(), desc)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get OCI image spec: %w", err)
+		return nil, fmt.Errorf("read image config from content store: %w", err)
 	}
+	var ociimage imagespec.Image
+	if err := json.Unmarshal(rb, &ociimage); err != nil {
+		return nil, fmt.Errorf("unmarshal image config %s: %w", rb, err)
+	}
+
+	// imageExtended is a superset of the oci Image struct. It helps
+	// to deserialize `ArgsEscaped` which is a non-OCI field that is
+	// used by docker to determine how the image command should be
+	// interpreted
+	type imageExtended struct {
+		Config struct {
+			ArgsEscaped bool `json:"ArgsEscaped,omitempty"`
+		}
+	}
+	// Deserialize the extended image format for Windows.
+	var ociImageExtended imageExtended
+	if err := json.Unmarshal(rb, &ociImageExtended); err != nil {
+		return nil, errors.Wrapf(err, "unmarshal image config for ArgsEscaped %s", rb)
```

Comment:
pkg/errors was removed as a dependency in this repo so best not to reintroduce it. Can use the built in wrapping support in Go.
```suggestion
		return nil, fmt.Errorf("unmarshal image config for ArgsEscaped %s: %w", rb, err)
```